config/settings/central.py

```
@@ -0,0 +1,3 @@
+"""Central node settings alias mapping to ``endo_api.settings_central``."""
+
+from endo_api.settings_central import *  # noqa: F401,F403
```

Ruff (0.14.0)

Unused `noqa` directive (non-enabled: `F401`)

Remove unused `noqa` directive

(RUF100)